test/_helpers.js

```diff
@@ -2,17 +2,26 @@ import { createGenerator } from '@unocss/core';
 import { beforeEach } from 'vitest';
 import { presetWarp } from '#plugin';
 
-export const getGenerator = (opts = {}) => createGenerator({ presets: [presetWarp({ ...opts, development: true })] });
+export const getGenerator = async (opts = {}) => {
+  const asyncPreset = await presetWarp({
+    ...opts,
+    development: true,
+  });
+  return createGenerator({ presets: [asyncPreset] });
+};
 export const setup = (opts = {}) => {
-  beforeEach(t => {
-    t.uno = getGenerator({ ...opts, development: true });
+  beforeEach(async (t) => {
+    t.uno = await getGenerator({
+      ...opts,
+      development: true,
```

Comment:
It was there before but now I wonder if we need to pass `development: true` again if the `getGenerator` already sets that option.

Reply:
True, I haven't thought about this one. Good catch. We can safely remove this one